google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfigurationPanel.java

```diff
@@ -123,6 +128,12 @@ public void applyEditorTo(@NotNull AppEngineDeploymentConfiguration configuratio
     configuration.setPromote(promoteCheckbox.isSelected());
     configuration.setCloudProjectName(projectSelector.getText());
     configuration.setStopPreviousVersion(stopPreviousVersionCheckbox.isSelected());
+    configuration.setDeployAllConfigs(deployAllConfigsCheckbox.isSelected());
```

Comment:
yea good point. I forgot that the ui element lived in the common panel (this was in anticipation of implementing it for flex as well - but this was delayed). I'm fine with it as you have it.